src/index.js

```diff
@@ -1,12 +1,14 @@
 import 'babel-polyfill';
 import 'source-map-support/register';
 
-import debug from 'debug';
+import debugLog from 'debug';
 import fs from 'graceful-fs';
 import mkdirp from 'mkdirp';
```

Comment:
it looks to be a logging thing. I see in the code below, debug('examining path %s', directoryPath);

Do we want to use the winston logger like we do in the other projects? How do other open source projects log things generally?